src/wx/widgets/sdljoy.cpp

```diff
@@ -314,24 +310,44 @@ void wxSDLJoy::Poll()
 
 void wxSDLJoy::ConnectController(uint8_t joy)
```

Comment:
I suggest renaming joy to joystick_index where appropriate.  This will help us differentiate and spot bugs more easily.  And also we should rename joy to something like SDL_JoystickID joystick_instance_id when it represents the instance_id.